client/jest.config.json

```diff
@@ -29,5 +29,8 @@
   "setupFilesAfterEnv": [
     "<rootDir>/test/unitTests/config/test.env.js",
     "<rootDir>/test/unitTests/config/importJestDom.ts"
-  ]
+  ],
+  "moduleNameMapper": {
+    "\\.css$": "<rootDir>/__mocks__/styleMock.js"
```

Comment:
why do we need this?

Reply:
This is needed for testing Signin because we are importing css styling for the GitLab button and without this we get errors in Signin.test.